orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/WaitForClusterDisableTask.groovy

```diff
@@ -16,14 +16,33 @@
 
 package com.netflix.spinnaker.orca.clouddriver.tasks.cluster
 
+import com.netflix.spinnaker.orca.DefaultTaskResult
+import com.netflix.spinnaker.orca.ExecutionStatus
+import com.netflix.spinnaker.orca.TaskResult
 import com.netflix.spinnaker.orca.clouddriver.pipeline.servergroup.support.TargetServerGroup
 import com.netflix.spinnaker.orca.clouddriver.utils.HealthHelper
+import com.netflix.spinnaker.orca.pipeline.model.Stage
 import org.springframework.stereotype.Component
 
 import java.util.function.Function
 
 @Component
 class WaitForClusterDisableTask extends AbstractWaitForClusterWideClouddriverTask {
+  private static final int MINIMUM_WAIT_TIME_MS = 90000
```

Comment:
Not a huge fan of the static wait ... but the current behavior has isServerGroupOperationInProgress completing as soon as the server group is disabled (_vs. waiting for health providers to go DOWN_). 

This smells like a bug but at the very least a small wait should account for any connection draining / discovery activity to complete.